articles/azure-arc/servers/overview.md

```diff
@@ -29,7 +28,9 @@ When you connect your machine to Azure Arc enabled servers, it enables the abili
     > [!NOTE]
     > At this time, enabling Update Management directly from an Arc enabled server is not supported. See [Enable Update Management from your Automation account](../../automation/update-management/enable-from-automation-account.md) to understand requirements and how to enable for your server.
 
-- Include your non-Azure servers for threat detection and proactively monitor for potential security threats using [Azure Security Center](../../security-center/security-center-introduction.md).
+- Include your non-Azure servers for advanced threat detection and proactively monitor for potential security threats using [Azure Security Center](../../security-center/security-center-introduction.md) or [Azure Defender](../../security-center/azure-defender.md).
+
+- Protect non-Azure servers with [Microsoft Defender for Endpoint](https://docs.microsoft.com/microsoft-365/security/defender-endpoint/microsoft-defender-endpoint), included through [Azure Defender](../../security-center/azure-defender.md), for threat detection, for vulnerability management, and to proactively monitor for potential security threats.
```

Comment:
Need to change the link for Microsoft Defender for Endpoint article from a URL to a referral - (/microsoft-365/security/defender-endpoint/microsoft-defender-endpoint).

Reply:
the referral is not valid since this is in the azure-docs area

Reply:
Site-relative links are available to support content hosted on docs.microsoft.com that exist in other repos/docsets. This functionality in the publishing platform supports this change I'm asking you to make. Thanks.